src/main/java/io/vertx/core/eventbus/impl/clustered/ClusteredMessage.java

```diff
@@ -245,4 +245,8 @@ ServerID getSender() {
   public boolean isFromWire() {
     return fromWire;
   }
+
+  boolean isLocal() {
+    return isFromWire();
```

Comment:
good catch, it would not have introduced a bug in eventbus, rather reporting incorrectly